app/src/main/java/com/github/se/polyfit/ui/screen/GraphScreen.kt

```diff
@@ -34,26 +34,30 @@ import com.github.se.polyfit.R
 import com.github.se.polyfit.ui.components.DropDownMenu
 import com.github.se.polyfit.ui.components.lineChartData
 import com.github.se.polyfit.ui.components.scaffold.SimpleTopBar
-import com.github.se.polyfit.ui.viewModel.DataToPoints
-import com.github.se.polyfit.ui.viewModel.DateList
+import com.github.se.polyfit.ui.viewModel.DisplayScreen
 import com.github.se.polyfit.ui.viewModel.GraphViewModel
 import com.github.se.polyfit.ui.viewModel.SortPoints
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun FullGraphScreen(viewModel: GraphViewModel = hiltViewModel<GraphViewModel>()) {
+fun FullGraphScreen(
+    viewModel: GraphViewModel = hiltViewModel<GraphViewModel>(),
+    goBack: () -> Unit
```

Comment:
Typically it's best to have mandatory parameters come before option ones
```suggestion
    goBack: () -> Unit,
    viewModel: GraphViewModel = hiltViewModel<GraphViewModel>(),
```